psycodict/database.py

```diff
@@ -662,8 +613,8 @@ def create_table(
         """
         if name in self.tablenames:
             raise ValueError("%s already exists" % name)
-        #if "_" not in name:
-        #    raise ValueError("Table name must contain an underscore; first part gives LMFDB section")
+        if "_" not in name:
+            raise ValueError("Table name must contain an underscore; first part gives LMFDB section")
```

Comment:
This was commented out in psycodict on purpose....

Reply:
I will try to move this into lmfdb, as we really want to files to match and not have remember subtle differences likes this